todo_app/templates/index.html

```diff
@@ -7,11 +7,19 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <from action="add Todo" method="post">
+    <label for="name">Enter New Todo: </label>
+    <input type="text" name="title" required />
+    <input type="submit" value="Add New Todo" />
+    </from>
```

Comment:
Watch out for typos here - it took me a few minutes to track down why this wasn't behaving as I'd expect! (and note that the `action` needs to be the path for the endpoint we're trying to hit, rather than its name or similar)
```suggestion
  <form action="add_new_items" method="post">
    <label for="name">Enter New Todo: </label>
    <input type="text" name="title" required />
    <input type="submit" value="Add New Todo" />
    </form>
```